gdpr_info/credit.mkd

```diff
@@ -0,0 +1,61 @@
+The final push to make the GDPR version of the open brain consent took place during a GLiMR workshop ([COST](https://www.cost.eu/) action CA18206) the 27th Novembre 2019, COST Association, 149 Avenue Louise, Brussels.
```

Comment:
this file is .mkd while the rest of the materials/website is .rst.  Might be worth having it converted, and then added/referenced somewhere within text

Reply:
### done :-) although my table doesn't show in GitHub?